specification/web/resource-manager/Microsoft.Web/stable/2023-01-01/CommonDefinitions.json

```diff
@@ -3570,6 +3574,54 @@
         }
       }
     },
+    "DaprConfig": {
+      "description": "App Dapr configuration.",
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "description": "Boolean indicating if the Dapr side car is enabled",
+          "type": "boolean",
+          "default": false
+        },
+        "appId": {
+          "description": "Dapr application identifier",
+          "type": "string"
+        },
+        "appPort": {
+          "format": "int32",
+          "description": "Tells Dapr which port your application is listening on",
+          "type": "integer"
+        },
+        "httpReadBufferSize": {
+          "type": "integer",
+          "description": "Dapr max size of http header read buffer in KB to handle when sending multi-KB headers. Default is 65KB.",
+          "format": "int32"
+        },
+        "httpMaxRequestSize": {
+          "type": "integer",
+          "description": "Increasing max size of request body http servers parameter in MB to handle uploading of big files. Default is 4 MB.",
+          "format": "int32"
+        },
+        "logLevel": {
+          "description": "Sets the log level for the Dapr sidecar. Allowed values are debug, info, warn, error. Default is info.",
+          "enum": [
+            "info",
+            "debug",
+            "warn",
+            "error"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "logLevel",
```

Comment:
can you use another name stead of logLevel? It will be confused between here and https://github.com/Azure/azure-rest-api-specs/blob/bae83065f059394a3b880f0b21de00686c129a8c/specification/web/resource-manager/Microsoft.Web/stable/2023-01-01/WebApps.json#L23989. this is why js sdk generator failed in build.

Reply:
@kazrael2119   , is there a way to get the sdk generator successful without the modification of this name?The Dapr properties added in this PR are going to be translated to  https://github.com/Azure/azure-rest-api-specs/blob/0ab5469dc0d75594f5747493dcfe8774e22d728f/specification/app/resource-manager/Microsoft.App/stable/2022-10-01/ContainerApps.json#L686, while designing our first principle was to keep these payloads as same as possible.

Reply:
I don't think enum name will go into the http payloads if that's what you are referring to. 

if you are suggesting that those two properties should be the same in the SDK interface, and we use some kind of directive to make it generate correctly, it will still be only renamed in the Microsoft.Web SDK side, which makes them inconsistent with Microsoft.App.  still breaks the principal. 

Another question I am having is, how do you make sure client side are using the matched api version between Microsoft.Web and Microsoft.App, if the api versions are not in match, payloads inconsisitencies still exist no matter it's in the http payload or SDK interface.

Reply:
@kazrael2119  is your suggestion that we make the changes you mentioned in https://github.com/Azure/azure-rest-api-specs/pull/25230 in this PR?